emcc.py

```diff
@@ -487,6 +484,21 @@ def is_supported(f):
   return results
 
 
+def map_library_names(link_flags):
+  system_libs_map = system_libs.Library.get_usable_variations()
```

Comment:
This scans the entire system library tree, and looks like the result is not cached. So this change makes us do it a second time (as there was a single caller before AFAICT). Might be worth seeing if this is not slow.

Reply:
Done